rasa/core/policies/keras_policy.py

```diff
@@ -223,23 +218,22 @@ def continue_training(
         batch_size = kwargs.get("batch_size", 5)
         epochs = kwargs.get("epochs", 50)
```

Comment:
why not using `self.batch_size` etc here?

Reply:
see the comment in the code above

Reply:
which one? because it's one epoch more?

Reply:
Sorry, but I don't understand it. Could you explain it to me? Doesn't `continue_training` do the same as `train` just that it operates on a model which was already trained previously?

Reply:
Ok, I think I got it now 👍